geonode_mapstore_client/client/js/epics/gnsave.js

```diff
@@ -126,23 +128,33 @@ const SaveAPI = {
 };
 
 export const gnSaveContent = (action$, store) =>
-    action$.ofType(SAVE_CONTENT)
+    action$.ofType(SAVE_CONTENT, TOGGLE_CONTROL)
```

Comment:
I would prefer if we use a separate epic to manage the toggle control or directly trigger the save content from the menu item of burger menu